### PR TITLE
rest-assured guide

### PR DESCRIPTION
**Pull request by sdelamo**

### CODE REVIEW

File: guides/micronaut-rest-assured/metadata.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Using Rest-Assured in Micronaut application",
```
Comment (burtbeckwith) [inline guides/micronaut-rest-assured/metadata.json]: "in Micronaut application" should be with "in a Micronaut application" or "in Micronaut applications"

File: guides/micronaut-rest-assured/metadata.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "base": "hello-base",
+  "intro": "Learn how to use use Rest-Assured to tests the REST API of your Micronaut Application",
```
Comment (burtbeckwith) [inline guides/micronaut-rest-assured/metadata.json]: tests -> test

File: guides/micronaut-rest-assured/micronaut-rest-assured.adoc [hunk 3]
```diff
@@ -0,0 +1,40 @@
+common:header.adoc[]
+
+== REST-assured
+
+In this guide, you test a Micronaut application using https://rest-assured.io[REST-assured]
```
Comment (burtbeckwith) [inline guides/micronaut-rest-assured/micronaut-rest-assured.adoc]: "you test" -> "you will test"

File: guides/micronaut-rest-assured/java/src/test/java/example/micronaut/HelloControllerTest.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+@MicronautTest // <1>
+public class HelloControllerTest {
+
+    @Inject
+    EmbeddedServer embeddedServer; // <2>
+
+    @Test
+    public void testHelloEndpoint() {
+        given()
```
Comment (burtbeckwith) [inline guides/micronaut-rest-assured/java/src/test/java/example/micronaut/HelloControllerTest.java]: I'd format this like the example at rest-assured.io to clearly separate the given/when/then sections

**Review by burtbeckwith — APPROVED** [review]
the content is rather sparse - maybe add more explanation and/or examples?

should add some content describing why this is a Java-focused guide - briefly contrast with Spock and Kotest

File: buildSrc/src/main/resources/pom.xml [hunk 5]
```diff
@@ -21,6 +21,11 @@
             <groupId>org.testcontainers</groupId>
             <artifactId>oracle-xe</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
+            <version>4.5.1</version>
```
Comment (burtbeckwith) [inline buildSrc/src/main/resources/pom.xml]: the latest is v5.1.0, is there an issue with using v5.x with Micronaut? If so, this should be described in the guide

File: guides/micronaut-rest-assured/micronaut-rest-assured.adoc (hunk 3 above)
Comment (sdelamo) [reply guides/micronaut-rest-assured/micronaut-rest-assured.adoc]: resolved via https://github.com/micronaut-projects/micronaut-guides/pull/900/commits/07303717ab9b664add0b9e8526f562dd6123103e

File: buildSrc/src/main/resources/pom.xml (hunk 5 above)
Comment (sdelamo) [reply buildSrc/src/main/resources/pom.xml]: resolved via https://github.com/micronaut-projects/micronaut-guides/pull/900/commits/fb9f33fea70fa0f24a8abad8a6a695655fbd089a

File: guides/micronaut-rest-assured/metadata.json (hunk 1 above)
Comment (sdelamo) [reply guides/micronaut-rest-assured/metadata.json]: resolved via https://github.com/micronaut-projects/micronaut-guides/pull/900/commits/e963f4463fc4620db3fb2d3f52cf67b0c7bba219

File: guides/micronaut-rest-assured/metadata.json (hunk 2 above)
Comment (sdelamo) [reply guides/micronaut-rest-assured/metadata.json]: resolved via https://github.com/micronaut-projects/micronaut-guides/pull/900/commits/e963f4463fc4620db3fb2d3f52cf67b0c7bba219

File: guides/micronaut-rest-assured/java/src/test/java/example/micronaut/HelloControllerTest.java (hunk 4 above)
Comment (sdelamo) [reply guides/micronaut-rest-assured/java/src/test/java/example/micronaut/HelloControllerTest.java]: resolved via https://github.com/micronaut-projects/micronaut-guides/pull/900/commits/b889a6cd7f6f7776916365c48613487d251e2d7d

**Comment by sdelamo** [comment]
> the content is rather sparse - maybe add more explanation and/or examples?
> 
> should add some content describing why this is a Java-focused guide - briefly contrast with Spock and Kotest

My goal is to have a small guide in place to show people that it is easy to use REST assured with Micronaut. The only special thing is setting the port. I want people coming from Quarkus examples using REST Assured to know that it is possible here as well.

**Review by burtbeckwith — APPROVED** [review]

File: guides/micronaut-rest-assured/metadata.json [hunk 6]
```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Using REST-Assured in a Micronaut application",
+  "base": "hello-base",
+  "intro": "Learn how to use use REST-Assured to test the REST API of your Micronaut Application",
```
Comment (kroegerk) [inline guides/micronaut-rest-assured/metadata.json:4]: lowercase "application"

**Review by kroegerk — COMMENTED** [review]
One minor capitalization revision

File: guides/micronaut-rest-assured/metadata.json (hunk 6 above)
Comment (sdelamo) [reply guides/micronaut-rest-assured/metadata.json:4]: resolved via https://github.com/micronaut-projects/micronaut-guides/pull/970/files#diff-0519f109579b4ba49d445ec6ff8a2b057d52ee59c9a8642b39b1997e0bdff987

**Pull request merged**